diff --git a/index.js b/index.js
index 43c060a0002c4b1cd6c5e5185ee7fdbab53b0d8d..3642a2b81b732bd529809fa543d9dad1946e9823 100644
--- a/index.js
+++ b/index.js
@@ -4,7 +4,8 @@ import path from 'node:path';
 import {fileURLToPath} from 'node:url';
 import {promisify} from 'node:util';
 import childProcess from 'node:child_process';
-import fs, {constants as fsConstants} from 'node:fs/promises';
+import fs from 'node:fs/promises';
+import {constants as fsConstants} from 'node:fs';
 import {isWsl, powerShellPath} from 'wsl-utils';
 import defineLazyProperty from 'define-lazy-prop';
 import defaultBrowser from 'default-browser';
